src/qutip_tensornetwork/testing.py

```diff
@@ -0,0 +1,100 @@
+import numpy as np
+def assert_network_close(actual, desired, atol=0, rtol=1e-7):
+    """Asserts if the given two ``Networks`` are not close. For two
+    ``Networks`` to be close  there needs to be a one-to-one map between their
+    nodes such that the tensors that these nodes represent are close to each
+    other. These ``Networks`` must also have same structure regarding to its
+    connections.
+
+    Parameters:
+    -----------
+    actual : list of nodes
+        ``Network`` obtained.
+
+    desired : list of nodes
+        Desired ``Network``.
+
+    atol: float, optional
+        Relative tolerance.
+
+    rtol: float, optional
+        Absolute tolerance.
+
+    Raises
+    ------
+    AssertionError
+        If actual and desired do not represent the same map.
+    ValueError
+        If more than one node in the network is close to each other.
```

Comment:
Currently the function raises ValueError because is not able to deal with networks that have more than one similar node. I will change the ValueError to NotImplementedError to make it clear that this is a missing feature but we should me to use the package you mentioned for the task of comparing two networks.